Author: Sean McAvoy
Summary: disables requirement of tests as they are not used.
----
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -19,7 +19,7 @@ include_directories (${CMAKE_CURRENT_BIN

 add_subdirectory(lib)
 add_subdirectory(cmd)
-add_subdirectory(test)
+#add_subdirectory(test)

 if (BARRIER_BUILD_GUI)
     add_subdirectory(gui)
